jerry-core/ecma/base/ecma-globals.h

```diff
@@ -204,6 +204,16 @@ typedef struct
   void *info_p; /**< free info or callback */
 } ecma_native_pointer_t;
 
+#ifdef JERRY_VM_EXEC_STOP
+
+/**
+ * Callback which returns with a non-undefined value
+ * when the ECMAScript execution should be stopped.
+ */
+typedef ecma_value_t (*jerry_vm_exec_stop_cb_t) (void *user_p);
```

Comment:
I've tried to figure out why this typedef duplication is needed (i.e., `jerryscript.h` also has `jerry_vm_exec_stop_callback_t`) but didn't succeed. We have other callback types in `jerryscript.h`, too (e.g., `jerry_external_handler_t`), but they have no duplicates internally. Couldn't we drop this also and use the single public typedef?